runtime/altair/src/lib.rs

```diff
@@ -1965,6 +1977,9 @@ impl_runtime_apis! {
 		fn metadata() -> OpaqueMetadata {
 			OpaqueMetadata::new(Runtime::metadata().into())
 		}
+
+		fn metadata_at_version(_: u32) -> Option<sp_core::OpaqueMetadata> { todo!("nuno") }
+		fn metadata_versions() -> frame_benchmarking::Vec<u32> { todo!("nuno") }
```

Comment:
**CR**

We must have something here. This would otherwise allow to kill nodes by calling this api.

Reply:
Done